README.md

```diff
@@ -1 +1,50 @@
-# crypto-hub
\ No newline at end of file
+# REACT CAPSTONE
+This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Redux
+- React
+## Sample images
+![screenshot](./src/images/Screenshot1.png)
+![screenshot](./src/images/Screenshot2.png)
+
+
+
+## Live previewhttps://deploy-preview-6--sympho
+[preview](https://crypto-hub1111.herokuapp.com/)
+## Install
+
+To get a local copy up and running follow these simple example steps.
+- Copy this link
+  ```
```

Comment:
- [ ] There should be a recorded video presenting the features of the project.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, highlighting all the project features.
  - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.